src/components/User/newUserPopup/index.js

```diff
@@ -0,0 +1,184 @@
+import React, { useMemo, useCallback } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { CSSTransition } from 'react-transition-group';
+import Input from '../../shared/Input';
+import { preventDefaultHandler } from '../../../utils';
+import NewUserCaption from './NewUserCaption';
+import styles from '../../shared/BasePopup/base.popup.module.scss';
+import stylesInput from '../../shared/Input/textField.module.scss';
+import '../../shared/BasePopup/styles.scss';
+import AlertMessage from '../../shared/AlertMessage';
+import PopupButtons from '../../shared/BasePopup/components/PopupButtons';
+import Textarea from '../../shared/Textarea';
+import {
+  getUserType,
+  getUsername,
+  getPassword,
+  getPasswordRepeat,
+  getUserDescription,
+  getErrorMessage,
+  getVisible,
+  isError,
+} from '../../../store/adminUsers/selectors';
+import {
+  newUser,
+  getUserTypeValue,
+  getUsernameValue,
+  getPasswordValue,
+  getPasswordRepeatValue,
+  getUserDescriptionValue,
+  onCloseAction,
+} from '../../../store/adminUsers/actions';
+
+const NewUserForm = ({
+  formUsername,
+  formPassword,
+  formPasswordRepeat,
+  formUserType,
+  formDescription,
+  isCreatingInProgress,
+  isVisible,
+  changePassword,
+  changeUsername,
+  changePasswordRepeat,
+  changeUserType,
+  changeUserDescription,
+  submitForm,
+  errorMessage,
+  isError,
+  okButton,
+  okButtonType,
+  cancelButton,
+  onClose,
+}) => {
+  const formHandler = useMemo(
+    () => preventDefaultHandler(submitForm),
+    [submitForm]
+  );
+  const alertClassesPassLen = useMemo(() => [
+    errorMessage === 'Password must be 6 or mor characters' ? styles.visible : styles.hidden,
```

Comment:
Not the best idea to compare error with string
Move it to some constant, mb, it is already there?